Wdb 50/create and setup react router

kibartas opened the pull request:
Nothing fancy

Commit:
[WDB-50]
add eslint rule so we can use the class based components in peace (thanks Aurimas >:( )

Commit:
[WDB-50]
create mocks for every page

Commit:
[WDB-50]
implement React router in App.js

tmaciulis22 commented on frontend/src/App.js:
```diff
@@ -1,20 +1,41 @@
+import TrashBinPage from './containers/TrashBinPage';
+import ProfilePage from './containers/ProfilePage';
+import ForgotPasswordPage from './containers/ForgotPasswordPage';
+import ResetPasswordPage from './containers/ResetPasswordPage';
```
Add confirm password page: https://zpl.io/bzQQ4xG

tmaciulis22 reviewed (approved):
Everything looks fine, just add additional page.

Commit:
Merge branch 'master' into WDB-50/Create-and-setup-react-router

Commit:
[WDB-50]
leave only async/await in the Api examples; change let values to const; add export default to every API file

Commit:
[WDB-50]
add redirection; check if user has JWT token in sessionStorage for access to some of the pages; add ConfirmEmailPage to Router

AArlauskas reviewed (approved):
LGTM
